tests/cases/setup_hammerdb/vars.json

```diff
@@ -1,5 +1,5 @@
 {
   "use_hostname": false,
-  "hammerdb": "/home/hammerdb/HammerDB-4.6",
-  "hammerdb_version": "4.6"
+  "hammerdb": "/home/hammerdb/HammerDB-4.7",
```

Comment:
I don't think we should force an upgrade at this time because we're in the middle of testing with a specific version.

But I do still see room for improvement.  This path should be based on whatever the hammerdb_version is.

This would also allow us to test with 4.7 before making a chance in default version.

Reply:
I can make that change.

I was updating because I rewatched the Benchmark KT meeting and Julien mentioned he wasn't using edb-ansible because it didn't have the latest version of HammerDB so thought this was needed.